config/ingest.py

```diff
@@ -19,11 +21,13 @@
 
 config.register.unique = ['visit']
 config.register.columns = {
-    'visit': 'int',
-    'basename': 'text',
-    'filter': 'text',
-    'date': 'text',
-    'expTime': 'double',
-    'object': 'text',
+    'visit':      'int',
+    'basename':   'text',
+    'filter':     'text',
+    'date':       'text',
+    'expTime':    'double',
+    'object':     'text',
+    'imgType':    'text',
+    'wavelength': 'double',
 }
```

Comment:
In general I find it best to _not_ align dictionary keys. If another key is added in the future that is longer than any of the current keys then you have to realign everything. But I don't think that our style guide or PEP8 forbids this, so feel free to do as you choose...